tests/test_modeling_common.py

```diff
@@ -554,6 +554,9 @@ def test_training_gradient_checkpointing(self):
             loss.backward()
 
     def test_attention_outputs(self):
+        if not self.has_attentions:
+            return
```

Comment:
```suggestion
            self.skipTest(reason="This model does not output attentions")
```

Reply:
We can probably get the model class here, and use f-string to make the reason more explicit. I am a bit lazy here though.

Reply:
Ok, not that there are many other tests just using "return", so we can update those as well

Reply:
We can not really get the model class, only the model tester

Reply:
We can: `BertModelTest` -> `BertModel`, or even look into `self.all_model_classes`. There will definitely a few edge cases to deal with though.

Reply:
Ok!